src/utils.js

```diff
@@ -65,13 +65,28 @@ module.exports.childSchemaEqual = function(a, b) {
  * @param {ANode} entity - Entity to begin the search on
  * @returns {ANode} An entity with a `networked` component or null
  */
-module.exports.getNetworkedEntity = function(entity) {
+function getNetworkedEntity(entity) {
   while(entity && !(entity.components && entity.components.networked)) {
     entity = entity.parentNode;
   }
   return entity;
 };
 
+module.exports.getNetworkedEntity = getNetworkedEntity;
+
+module.exports.takeOwnership = function(entity) {
+  const networkedEntity = getNetworkedEntity(entity);
+  if(!networkedEntity) {
```

Comment:
This should log an error similar to https://github.com/networked-aframe/networked-aframe/blob/master/src/components/networked-audio-source.js#L24